pymeasure/instruments/keithley/keithley6221.py

```diff
@@ -93,6 +93,16 @@ def __init__(self, adapter, name="Keithley 6221 SourceMeter", **kwargs):
         map_values=True
     )
 
+    shield_connection = Instrument.control(
+        ":OUTPut:ISHield?", ":OUTPut:ISHield %s",
+        """ A string property that controls
+        to whom the shield is connected, default to 'output-low'.
+        Valid values are 'guard' and 'output-low'. """,
```

Comment:
Please follow this new docstring style: https://pymeasure.readthedocs.io/en/latest/dev/coding_standards.html#docstrings